datasets/farsi_news/farsi_news.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 the HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Farsi News Datasets: Hamshahri and RadioFarda"""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+
+import datasets
+
+
+_CITATION = """\
+"""
+
+_DESCRIPTION = """\
+
```

Comment:
can you add a few lines describing the dataset here ?

and also add the citation if there's one ?

Reply:
There's no citation (only a github link, what should I do ?)

Reply:
let's keep it empty then